src/ingestion-server/server/images/nginx/config/nginx.conf

```diff
@@ -75,6 +75,34 @@ http {
             proxy_pass http://127.0.0.1:8685;
         }
 
+        location %%PING_ENDPOINT_PATH%% {
+            access_log off;
+
+            add_header Content-Type text/plain;
```

Comment:
Let's use JSON for supporting other attributes in the future.

Reply:
Fixed